controllers/ocsinitialization/sccs.go

```diff
@@ -103,44 +104,13 @@ func newRookCephSCC(namespace string) *secv1.SecurityContextConstraints {
 }
 
 func newRookCephCSISCC(namespace string) *secv1.SecurityContextConstraints {
-	scc := blankSCC()
-
-	scc.Name = "rook-ceph-csi"
-	scc.AllowPrivilegedContainer = true
-	scc.AllowHostNetwork = true
-	scc.AllowHostDirVolumePlugin = true
-	scc.AllowedCapabilities = []corev1.Capability{
-		secv1.AllowAllCapabilities,
-	}
-	scc.AllowHostPorts = true
-	scc.AllowHostPID = true
-	scc.AllowHostIPC = true
-	scc.ReadOnlyRootFilesystem = false
-	scc.RequiredDropCapabilities = []corev1.Capability{}
-	scc.DefaultAddCapabilities = []corev1.Capability{}
-	scc.RunAsUser = secv1.RunAsUserStrategyOptions{
-		Type: secv1.RunAsUserStrategyRunAsAny,
-	}
-	scc.SELinuxContext = secv1.SELinuxContextStrategyOptions{
-		Type: secv1.SELinuxStrategyRunAsAny,
-	}
-	scc.FSGroup = secv1.FSGroupStrategyOptions{
-		Type: secv1.FSGroupStrategyRunAsAny,
-	}
-	scc.SupplementalGroups = secv1.SupplementalGroupsStrategyOptions{
-		Type: secv1.SupplementalGroupsStrategyRunAsAny,
-	}
-	scc.Volumes = []secv1.FSType{
-		secv1.FSTypeAll,
-	}
-	scc.Users = []string{
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-rbd-plugin-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-rbd-provisioner-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-rbd-attacher-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-cephfs-plugin-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-cephfs-provisioner-sa", namespace),
+	rookValues := cephcsi.SecurityContextConstraintsValues{
+		Namespace: "rook-ceph",
```

Comment:
need to use `namespace` input parameter as the namespace will be openshift-storage

Reply:
Oh, right, of course!